public/config/configuration.json

```diff
@@ -7,6 +7,7 @@
   "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
   "BUSINESS_CREATE_LD_CLIENT_ID": "5e6142dd29b3030a850285b3",
   "DASHBOARD_URL": "https://dev.bcregistry.ca/business/",
+  "HOTJAR_ID": "3274073",
```

Comment:
When I figure out how Hotjar/surveys can work for our purposes, then I will store this key in 1Password (and probably leave it blank here for local development).

At the moment, I don't know what's possible or what it looks like, and the design team needs to weigh is as well. So this is a real experiment, haha.

I'm not even sure that we will kick off surveys from here. We might want to kick them off from Filings UI after returning from an IA filing/payment.